packages/components/psammead-episode-list/src/index.jsx

```diff
@@ -34,6 +34,16 @@ const StyledEpisodeListItem = styled.li`
   }
 `;
 
+const VerticalSeparator = styled.span`
+  padding-left: 8px;
+  margin-right: 8px;
+  border-right: 1px solid ${C_CLOUD_LIGHT};
```

Comment:
This probably looks better if users resize text:
```suggestion
  padding-left: 0.5rem;
  margin-right: 0.5rem;
  border-right: 0.0625rem solid ${C_CLOUD_LIGHT};
```
(0.5rem could be the `GEL_SPACING` const instead)